content/telegraf/v1.21/about_the_project/release-notes-changelog.md

```diff
@@ -10,6 +10,51 @@ menu:
     weight: 10
     parent: About the project
 ---
+## v1.21.3 [2022-1-31]
```

Comment:
Can we set the date to [2022-1-27]? I wanna keep it aligned with what's in the GitHub Changelog